boards/arm/samv7/same70-xplained/Kconfig

```diff
@@ -90,4 +90,38 @@ config SAME70XPLAINED_HSMCI0_AUTOMOUNT_UDELAY
 
 endif # SAME70XPLAINED_HSMCI0_AUTOMOUNT
 
+menuconfig SAME70XPLAINED_APP_FORMAT_MCUBOOT
+	bool "MCUboot-bootable format"
+	default n || MCUBOOT_BOOTLOADER
```

Comment:
Is this correct?

Reply:
In general when "MCUBOOT_BOOTLOADER" is enabled then it needs access to OTA partitions.
I think that my idea was to use `#if defined(CONFIG_SAME70XPLAINED_APP_FORMAT_MCUBOOT)` only instead of `#if defined(CONFIG_MCUBOOT_BOOTLOADER) || defined(CONFIG_SAME70XPLAINED_APP_FORMAT_MCUBOOT)`, but seems that I didn't finalized it.
I will double check and come back with some comments

Reply:
`MCUBOOT_BOOTLOADER` being a config from the APPS repository should not be referenced on the OS repository.
Conceptually speaking, the generation of the NuttX binary in MCUboot format does not depend on the MCUboot library or application being integrated.

Reply:
Ok. I will redo and separate those parts. `CONFIG_SAME70XPLAINED_APP_FORMAT_MCUBOOT` will only be an option for linker file selection and probably I will add additional option like `CONFIG_SAME70XPLAINED_HAVE_OTA_PARTITION`.
But then comes a question if I can still use `#if defined(CONFIG_MCUBOOT_BOOTLOADER)` in board specific files because it is an application level config?

Reply:
Can I still use `depends on BOOT_MCUBOOT` in OS repository? Or those should be completely separated?

Reply:
Done.